e2etest/manifests/mirror-server.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: mirror
-        image: metallb/mirror-server:master
+        image: metallb/mirror-server:main
```

Comment:
Not sure I follow the commit message here (just in case, as GH really doesn't help with this is the commit `s/master/main/g in mirror-server.yaml`).

Sorry if it's a trivial question (haven't tried the inv script yet), but does it generate the `metallb/mirror-server:main` image locally and, with this patch, it doesn't reach for dockerhub to get it? Because the `metallb/mirror-server:main` surely exist on dockerhub

Reply:
This is fallout from some changes done a while ago following switching from `master` to `main`. The commits which introduced the changes are mentioned in my commit message. What I've tried to do here is to get the dev environment working again by adding a missing rename of `master` to `main` in `mirror-server.yaml`.

`inv dev-env` builds the Go code, generates Docker images for the controller and the speaker, uploads the images to the [kind](https://github.com/kubernetes-sigs/kind/) cluster and applies the manifests while setting `imagePullPolicy` to `Never`. This ensures you always run your dev version on the dev environment.

Reply:
Ohh, thanks for the explanation! :)